titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/workflow/step/MustBeRelocatedTaskStoreUpdateStep.java

```diff
@@ -119,7 +123,7 @@ private void updateInStore(List<TaskRelocationPlan> toUpdate) {
         Map<String, Optional<Throwable>> result;
         try {
             result = store.createOrUpdateTaskRelocationPlans(toUpdate)
-                    .timeout(STORE_UPDATE_TIMEOUT)
+                    .timeout(Duration.ofMillis(configuration.getRdsTimeoutMs()))
```

Comment:
if you timeout the Mono operation, does the underlying sql query also get cancelled or is the query just going to run until completion?

Reply:
I think it will not be cancelled. This request is run synchronously on an internal executor.